ouroboros-consensus/src/Ouroboros/Consensus/Storage/ChainDB/Impl/Background.hs

```diff
@@ -98,9 +92,8 @@ launchBgTasks cdb@CDB{..} replayed = do
       gcScheduleRunner gcSchedule $ garbageCollect cdb
     !copyAndSnapshotThread <- launch "ChainDB.copyAndSnapshotRunner" $
       copyAndSnapshotRunner cdb gcSchedule replayed
-    !chainSyncThread <- scheduledChainSelectionRunner cdb
```

Comment:
Which idiot ever came up with the name for that thread? It has nothing to do with chain sync! :smirk:

Reply:
Some people... :shrug: